From eb08b01773fa7d2ca47cd1b652bf23661ead0b94 Mon Sep 17 00:00:00 2001
From: Myy <myy@miouyouyou.fr>
Date: Sun, 13 Nov 2016 08:22:09 +0000
Subject: [PATCH 3/3] Post Mali Kernel device drivers modifications.

Once the "drivers/gpu/arm" folder of the Mali Kernel device drivers
copied in drivers/gpu, you can apply this patch to enable the
compilation of the Mali Kernel device drivers.

The Mali Kernel device drivers :
http://malideveloper.arm.com/resources/drivers/open-source-mali-midgard-gpu-kernel-drivers/

You'll still need appropriate user-space libraries to exploit the 3D
functionnalities of your hardware. Currently, that only includes the
closed-source proprietary Mali user-space drivers :
http://malideveloper.arm.com/resources/drivers/arm-mali-midgard-gpu-user-space-drivers/

Signed-off-by: Myy <myy@miouyouyou.fr>
---
 drivers/gpu/Makefile  | 2 +-
 drivers/video/Kconfig | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/Makefile b/drivers/gpu/Makefile
index e9ed439..66386b4 100644
--- a/drivers/gpu/Makefile
+++ b/drivers/gpu/Makefile
@@ -2,5 +2,5 @@
 # taken to initialize them in the correct order. Link order is the only way
 # to ensure this currently.
 obj-$(CONFIG_TEGRA_HOST1X)	+= host1x/
-obj-y			+= drm/ vga/
+obj-y			+= drm/ vga/ arm/
 obj-$(CONFIG_IMX_IPUV3_CORE)	+= ipu-v3/
diff --git a/drivers/video/Kconfig b/drivers/video/Kconfig
index 3c20af9..041e15f 100644
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -17,6 +17,7 @@ source "drivers/gpu/vga/Kconfig"
 
 source "drivers/gpu/host1x/Kconfig"
 source "drivers/gpu/ipu-v3/Kconfig"
+source "drivers/gpu/arm/midgard/Kconfig"
 
 source "drivers/gpu/drm/Kconfig"
 
-- 
2.4.10

